packages/x-date-pickers/src/ClockPicker/ClockPicker.test.tsx

```diff
@@ -194,7 +194,10 @@ describe('<ClockPicker />', () => {
     expect(Math.max(...hours)).to.equal(23);
   });
 
-  it('should display options, but not update value when readOnly prop is passed', () => {
+  it('should display options, but not update value when readOnly prop is passed', function test() {
+    if (/safari/i.test(window.navigator.userAgent) || /firefox/i.test(window.navigator.userAgent)) {
+      this.skip();
+    }
```

Comment:
For each test, could we leave a comment on what's missing to make it work? I believe we have done this systematically in the past. 

For instance:
```suggestion
    // Only run in supported browsers
    if (typeof Touch === 'undefined') {
      this.skip();
    }
```

Reply:
👍 Sounds easier to read

Reply:
And more reliable, if tomorrow the API is available, then the test starts running.